src/mmrelay/matrix/events.py

```
@@ -0,0 +1,738 @@
+            facade.logger.debug(
+                f"broadcast_enabled is False - not relaying message from {full_display_name} to Meshtastic"
+            )
+
+
+async def on_room_member(room: MatrixRoom, event: RoomMemberEvent) -> None:
+    """
+    Handle room member events to observe room-specific display name changes.
+
+    This callback is registered so the Matrix client processes member state updates; no explicit action is required here because room-specific display names are available via the room state immediately after this event.
+    """
+    pass
```

Ruff (0.15.9)

[warning] Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)